assume/common/forecasts.py

```diff
@@ -127,6 +148,30 @@ def calc_forecast_if_needed(self):
                 "residual_load_EOM"
             ] = self.calculate_residual_demand_forecast()
 
+    def get_heating_demand(self) -> pd.Series:
+        """
+        Returns the heating demand of the building
+
+        :rtype: pd.Series
+        """
+        return self.forecasts.get("heating_demand", pd.Series(0, self.index))
+
+    def get_cooling_demand(self) -> pd.Series:
+        """
+        Returns the cooling demand of the building
+
+        :rtype: pd.Series
+        """
+        return self.forecasts.get("cooling_demand", pd.Series(0, self.index))
+
+    def get_eom_price_forcast(self) -> pd.Series:
+        """
+        Returns the heating demand of the building
+
+        :rtype: pd.Series
+        """
+        return self.forecasts.get("price_EOM", pd.Series(0, self.index))
+
     def get_registered_market_participants(self, market_id):
```

Comment:
this whole section seems redundant.. Doesnt the same happen if i run self["price_EOM"] for example? Why do I need the getter for this?